src/views/Entities/ExternalServices.vue

```diff
@@ -0,0 +1,381 @@
+    FrameSkeleton,
+    Pagination,
+    DataOverview,
+    Tabs,
+    YamlView,
+    LabelList
+  },
+  mixins: [
+    FormatForCLI,
+    sortEntities
+  ],
+  data () {
+    return {
+      isLoading: true,
+      isEmpty: false,
+      hasError: false,
+      entityIsLoading: true,
+      entityIsEmpty: false,
+      entityHasError: false,
+      tableDataIsEmpty: false,
+      empty_state: {
+        title: 'No Data',
+        message: 'There are no External Services present.'
+      },
+      tableData: {
+        headers: [
+          { key: 'actions', hideLabel: true },
+          { label: 'Name', key: 'name' },
+          { label: 'Mesh', key: 'mesh' },
+          { label: 'Address', key: 'address' },
+          { label: 'TLS', key: 'tlsEnabled' }
+        ],
+        data: []
+      },
+      tabs: [
+        {
+          hash: '#overview',
+          title: 'Overview'
+        },
+        {
+          hash: '#yaml',
+          title: 'YAML'
+        }
+      ],
+      entity: [],
+      rawEntity: null,
+      firstEntity: null,
+      pageSize: this.$pageSize,
+      pageOffset: null,
+      next: null,
+      hasNext: false,
+      previous: []
+    }
+  },
+  computed: {
+    tabGroupTitle () {
+      const entity = this.entity
+
+      if (entity) {
+        return `External Service: ${entity.name}`
+      } else {
+        return null
+      }
+    },
+    entityOverviewTitle () {
+      const entity = this.entity
+
+      if (entity) {
+        return `Entity Overview for ${entity.name}`
+      } else {
+        return null
+      }
+    },
+    formattedRawEntity () {
+      const entity = this.formatForCLI(this.rawEntity)
+
+      return entity
+    },
+    shareUrl () {
+      const urlRoot = `${window.location.origin}#`
+      const entity = this.entity
+
+      const shareUrl = () => {
+        if (this.$route.query.ns) {
+          return this.$route.fullPath
+        }
+
+        return `${urlRoot}${this.$route.fullPath}?ns=${entity.name}`
+      }
+
+      return shareUrl()
+    }
+  },
+  watch: {
+    '$route' (to, from) {
+      this.init()
+    }
+  },
+  beforeMount () {
+    this.init()
+  },
+  methods: {
+    init () {
+      this.loadData()
+    },
+    goToPreviousPage () {
+      this.pageOffset = this.previous.pop()
+      this.next = null
+
+      this.loadData()
+    },
+    goToNextPage () {
+      this.previous.push(this.pageOffset)
+      this.pageOffset = this.next
+      this.next = null
+
+      this.loadData()
+    },
+    tableAction (ev) {
+      const data = ev
+
+      // reset back to the first tab
+      this.$store.dispatch('updateSelectedTab', this.tabs[0].hash)
+
+      // set the active table row
+      this.$store.dispatch('updateSelectedTableRow', data.name)
+
+      // load the data into the tabs
+      this.getEntity(data)
+    },
+    loadData () {
+      this.isLoading = true
+
+      const mesh = this.$route.params.mesh || null
+      const query = this.$route.query.ns || null
```

Comment:
Is there a reason, you are assigning `null` values when these elems have falsy values? I don't see in this function strict checking for nulls